src/patternfly/layouts/Flex/flex.scss

```diff
@@ -312,4 +337,53 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
     }
   }
 }
+
+// .pf-m-gap-{size}{-on-breakpoint}
+@each $breakpoint, $breakpoint-value in $pf-l-flex--breakpoint-map {
+  $breakpoint-name: if($breakpoint != "base", -on-#{$breakpoint}, "");
```

Comment:
similar to `$breakpoint-name` here, if you have a conditional for `$spacer` that only includes the `-#{$spacer}` if the key isn't "base", you can remove the [classes that set `.pf-m-gap/row-gap/column-gap`](https://github.com/patternfly/patternfly/blob/a9848b35565359c9bf7f1dddd73078b3252c7ec9/src/patternfly/layouts/Flex/flex.scss#L170-L186) and move those into this block so it will generate `.pf-m-gap[-[sizes][-on-[breakpoints]]`

Reply:
done